modules/openapi-generator/src/main/resources/python/configuration.mustache

```diff
@@ -123,19 +152,21 @@ class Configuration(object):
         self.password = password
         """Password for HTTP basic authentication
         """
+{{#hasHttpSignatureMethods}}
         if signing_info is not None:
             signing_info.host = host
         self.signing_info = signing_info
         """The HTTP signing configuration
         """
+{{/hasHttpSignatureMethods}}
 {{#hasOAuthMethods}}
-        self.access_token = ""
+        self.access_token = None
         """access token for OAuth/Bearer
         """
 {{/hasOAuthMethods}}
 {{^hasOAuthMethods}}
 {{#hasBearerMethods}}
-        self.access_token = ""
+        self.access_token = None
```

Comment:
The above comment about targeting a different release or changing this back to empty string applies here.